src/main/java/org/openlmis/requisition/service/RequisitionService.java

```diff
@@ -304,7 +304,9 @@ public Requisition reject(UUID requisitionId) {
           requisition.getSupervisoryNodeId(), userId);
 
       LOGGER.debug("Requisition rejected: {}", requisitionId);
-      requisition.reject(orderableReferenceDataService.findAll(), userId);
```

Comment:
Reject took a list of orderables in order to do some calculations on line items - each line item has corresponding orderable and we couldn't search them based on facility/program; I guess it was still better than calling findOne for each item as we did in ReqDtoBuilder